l5kit/l5kit/geometry/transform.py

```diff
@@ -75,8 +75,8 @@ def flip_y_axis(tm: np.ndarray, y_dim_size: int) -> np.ndarray:
 def transform_points(points: np.ndarray, transf_matrix: np.ndarray) -> np.ndarray:
     """
     Transform points using transformation matrix.
-    Note this function assumes points.shape[1] == matrix.shape[1] - 1, which means that the last row on the matrix
-    does not influence the final result.
+    Note this function assumes points.shape[1] == matrix.shape[1] - 1, which means that the last
+    row on the matrix does not influence the final result.
```

Comment:
can we actually replace this "on" with "in"? I've put it there in the first place but now it sounds weird :)